src/map/pc.cpp

```diff
@@ -10250,11 +10250,6 @@ static int pc_checkcombo(struct map_session_data *sd, item_data *data) {
 
 		combo_idx.reserve(nb_itemCombo);
 
-		// Zero out temporary combo array
-		for (auto &tmp_combo : combo_idx) {
```

Comment:
add a constructor in s_itemchk, so it will be more safe and strickly equivalent.
bt